js/logic.js

```diff
@@ -0,0 +1,60 @@
+/* eslint-disable no-undef */
+/* eslint-disable no-unused-vars */
+/* eslint-disable no-use-before-define */
```

Comment:
Why?

Reply:
Having a file for DOM and another for the game logic implies instantiating variable in one but not both files at the time to prevent, clarifying this, it is necessary to disable the following rules to avoid linting validation issues.

/* eslint-disable no-undef */
/* eslint-disable no-use-before-define */

I hope this explanation suffices your concerns.